mkt/reviewers/tests/test_views.py

```diff
@@ -3392,6 +3393,101 @@ def test_deleted_app(self):
         eq_(doc('.tabnav li a')[0].text, u'Abuse Reports (1)')
 
 
+class TestWebsiteAbuseQueue(mkt.site.tests.TestCase, AccessMixin):
```

Comment:
I didn't think about that when you did the app abuse viewer, but can you add XSS tests ? To make sure we are properly escaping the data displayed in the queue etc.